components/bl-rating-reviews/src/index.js

```diff
@@ -0,0 +1,43 @@
+import { useMemo } from 'react';
+
+import { calculateAverage, calculatePercent, calculateTotalReviews, prepareReviewData } from './helpers';
+import { ViewsMap } from './views';
+
+const { cn, normalizeDimensionValue } = BackendlessUI.CSSUtils;
+
+export default function CustomUIComponent({ component, eventHandlers, elRef }) {
+  const { ratingData, color, iconSize, width, view, localizations, style, classList } = component;
+
+  const processedData = useMemo(() => prepareReviewData(ratingData, color), [ratingData, color]);
+  const maxValue = useMemo(() => processedData.length, [processedData.length]);
```

Comment:
Redundant action.

It's primitive, not object, so that doesn't prevent rerenders.